packages/graphback-codegen-schema/tests/GraphQLSchemaCreatorTest.ts

```diff
@@ -229,3 +229,50 @@ test('Creates CRUD resolvers for models', async () => {
 
   expect(metadata.getResolvers()).toMatchSnapshot();
 });
+
+test('field directives on relationship fields are mapped to schema', () => {
+  const schemaGenerator = new SchemaCRUDPlugin();
+  const modelAST = `directive @test on FIELD_DEFINITION
+
+  """ @model """
+type Note {
+  id: ID!
+  title: String!
+  description: String
+  """
+  @oneToMany(field: 'note')
+  """
+  comments: [Comment]! @test
+  """@oneToOne"""
+  comment: Comment @test
+}
+
+""" @model """
+type Comment {
+  id: ID!
+  text: String
+  description: String
+  note: Note! @test
+}`
+
+  const metadata = new GraphbackCoreMetadata({ crudMethods: {} }, buildSchema(modelAST));
+  const schema = schemaGenerator.transformSchema(metadata);
+
+  const noteType = schema.getType('Note') as GraphQLObjectType
+  const { comments, comment } = noteType.getFields()
+  
+  // Note.comments field
+  expect(comments.astNode?.directives).toHaveLength(1)
+  expect(comments.astNode?.directives[0].name.value).toBe('test')
+
+  // Note.comment field
+  expect(comment.astNode?.directives).toHaveLength(1)
+  expect(comment.astNode?.directives[0].name.value).toBe('test')
+
+  const commentType = schema.getType('Comment') as GraphQLObjectType
+  const { note } = commentType.getFields()
+
+  // Note.comments field
+  expect(note.astNode?.directives).toHaveLength(1)
+  expect(note.astNode?.directives[0].name.value).toBe('test')
```

Comment:
Thanks for adding these tests. 

Do you think we can have a snapshot test too?

Reply:
Sure, pushed now :wink: